index.html

```diff
@@ -11286,20 +11289,20 @@ <h2 id="AudioListener">
             </dt>
             <dd>
               <p>
-                This method is DEPRECATED. It is equivalent to setting
-                <code>positionX.value</code>, <code>positionY.value</code>, and
-                <code>positionZ.value</code> directly with the given
-                <code>x</code>, <code>y</code>, and <code>z</code> values,
-                respectively.
+                This method is DEPRECATED. It is equivalent to setting <a href=
+                "#dom-AudioListener-positionx">positionX.value</a>, <a href=
```

Comment:
Likewise, replace `<a href=...>` with `<a data-link-for="AudioListener">positionX</a><code>.value</code>`

Reply:
I'm ok if you want to land this now.  I'll fix any broken links later.